example/lib/board_editor_page.dart

```diff
@@ -0,0 +1,191 @@
+import 'package:board_example/board_theme.dart';
+import 'package:flutter/material.dart';
+import 'package:chessground/chessground.dart';
+import 'package:dartchess/dartchess.dart' as dc;
+import 'package:collection/collection.dart';
+
+class BoardEditorPage extends StatefulWidget {
+  const BoardEditorPage({super.key});
+
+  @override
+  State<StatefulWidget> createState() => _BoardEditorPageState();
+}
+
+class _BoardEditorPageState extends State<BoardEditorPage> {
+  Pieces pieces = readFen(dc.kInitialFEN);
+
+  dc.Piece? pieceToAddOnTouch;
+  bool deleteOnTouch = false;
```

Comment:
This is possibly redundant now. We could also decide to delete pieces when `dc.Piece` is `null`.